sql/dolphinscheduler_mysql.sql

```diff
@@ -703,7 +703,7 @@ CREATE TABLE `t_ds_task_instance` (
   `alert_flag` tinyint(4) DEFAULT NULL COMMENT 'whether alert',
   `retry_times` int(4) DEFAULT '0' COMMENT 'task retry times',
   `pid` int(4) DEFAULT NULL COMMENT 'pid of task',
-  `app_link` text COMMENT 'yarn app id',
+  `app_link` varchar(255) DEFAULT NULL COMMENT 'yarn app id',
```

Comment:
Hi,the type of `app_link` should be text,it was updated  when released the 1.3.0 version

Reply:
But in `1.3.5-release`, `app_link` is `varchar(255)` with link[https://github.com/apache/incubator-dolphinscheduler/blob/1.3.5-release/sql/dolphinscheduler_mysql.sql#L704](https://github.com/apache/incubator-dolphinscheduler/blob/1.3.5-release/sql/dolphinscheduler_mysql.sql#L704) I just keep it same as `1.3.5-release`. If `app_link` needs to be set to `text`, it's better in `1.3.6-release`, and the dev branch just merges the `1.3.6-release`

Reply:
Good job,May be the app_link field had been updated under the sql/upgrade,but was not updated  in sql/dolphinscheduler-xxx.sql.